frontend/src/components/common/StudentDetailCommon.vue

```diff
@@ -702,6 +804,22 @@ export default {
         this.spacePostalCode = this.studentCopy.postalCode.replace(/.{3}$/, ' $&');
       }
     },
+    setMinCodeLabel() {
+      if (this.studentCopy.mincode) {
+        this.minCodeLabel = formatMinCode(this.studentCopy.mincode);
+      }
+    },
+    setMinCode(minCodeLabel) {
+      if (minCodeLabel) {
+        if (minCodeLabel.includes(' ')) {
```

Comment:
`minCodeLabel.replace` should already checked if there's a space char, so I think we don't need `minCodeLabel.includes(' ')`

Reply:
agree.  I will update it.